lib/rails/auth/controller_methods.rb

```diff
@@ -0,0 +1,20 @@
+require "active_support/hash_with_indifferent_access"
+
+module Rails
+  module Auth
+    # Convenience methods designed to be included in an ActionController::Base subclass
+    # Recommended use: include in ApplicationController
+    module ControllerMethods
+      # Obtain credentials for the current request
+      #
+      # @return [HashWithIndifferentAccess] credentials extracted from the environment
+      #
+      def credentials
```

Comment:
what about calling these `ssl_credentials` or something? to differentiate from something like cookie/session credentials?